docs/api.md

```diff
@@ -321,6 +321,7 @@ back to the client. This code will be traded for a token at the
 - `access_type`: Optional. A value of `offline` will generate a refresh token along with the access token.
 - `code_challenge_method`: Required if using [PKCE](pkce.md). Must be `S256`, no other value is accepted.
 - `code_challenge`: Required if using [PKCE](pkce.md). A minimum length of 43 characters and a maximum length of 128 characters string, encoded as `BASE64URL`.
+- `keys_jwe`: A JWE bundle of the request.
```

Comment:
This could use a few more words; suggestion: "(Optional) A JWE bundle to be returned to the client when it redeems the authorization code`.